routes/web.php

```diff
@@ -3,6 +3,7 @@
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\HomeController;
 use App\Http\Controllers\MovieController;
+use App\Http\Models\Page;
```

Comment:
Import the PageController class that your using with the Route function instead of the Page model being imported here.